be/src/formats/parquet/stored_column_reader.cpp

```diff
@@ -227,11 +233,15 @@ Status RepeatedStoredColumnReader::read_records(size_t* num_records, ColumnConte
     return Status::OK();
 }
 
-Status RepeatedStoredColumnReader::_next_page() {
+// For repeated columns, it is difficult to skip rows according the num_values in header because
```

Comment:
we can use def level and rep level to calculate how many rows in this page

Reply:
Yes, it is feasible. But we need to read and decompress page data, then parse rep level to check whether select, I'm not sure whether it is worthy.